.pre-commit-config.yaml

```diff
@@ -0,0 +1,43 @@
+repos:
```

Comment:
Pitäiskö se markdownlint-cli hook lisätä kans, jos lisäsit tuon conventional-pre-commitinkin?

Reply:
Let's do this later but good idea